winpr/libwinpr/sspi/NTLM/ntlm.c

```diff
@@ -47,22 +48,17 @@ char* NTLM_PACKAGE_NAME = "NTLM";
 int ntlm_SetContextWorkstation(NTLM_CONTEXT* context, char* Workstation)
 {
 	int status;
-	DWORD nSize = 0;
+	DWORD nSize = MAX_COMPUTERNAME_LENGTH;
 	char* ws = Workstation;
+	TCHAR computerName[MAX_COMPUTERNAME_LENGTH + 1];
```

Comment:
Since you're explicitly using GetComputerNameExA and _strdup below you should not use TCHAR but CHAR here.